File: Snooper-Okhttp-No-Op/src/main/java/com/prateekj/snooper/okhttp/SnooperInterceptor.java
package com.prateekj.snooper.okhttp;

import java.io.IOException;

import okhttp3.Interceptor;
import okhttp3.Response;

public class SnooperInterceptor implements Interceptor {
  @Override
  public Response intercept(Chain chain) throws IOException {
    return chain.proceed(chain.request());
  }
}

File: Snooper-Okhttp-No-Op/src/main/java/com/prateekj/snooper/okhttp/SnooperInterceptor.kt
package com.prateekj.snooper.okhttp

import java.io.IOException

import okhttp3.Interceptor
import okhttp3.Response

class SnooperInterceptor : Interceptor {
  @Throws(IOException::class)
  override fun intercept(chain: Interceptor.Chain): Response {
    return chain.proceed(chain.request())
  }
}
